arbnode/seq_coordinator.go

```diff
@@ -789,6 +789,14 @@ func (c *SeqCoordinator) trySwitchingRedis(ctx context.Context) error {
 				err, "newRedisUrl", c.config.NewRedisUrl)
 			return err
 		}
+		err = c.wantsLockoutUpdate(ctx, c.Client)
+		if err != nil {
+			return err
+		}
+		err = c.wantsLockoutUpdate(ctx, newRedisCoordinator.Client)
```

Comment:
We want to write the "wants lockout" key to the new redis even before we notice the switchover. As-is, whoever's first to notice the switchover won't see anybody else's "wants lockout" key. We want all the "wants lockout" keys to be there before the switchover actually happens.